api/openapi-spec/swagger.json

```diff
@@ -3829,6 +3829,10 @@
      "name"
     ],
     "properties": {
+     "bootorder": {
+      "description": "Boot order for this interface.\nany number larger than zero gives the order in which this interface will be used\nfor bootting among all other devices (interfaces and disks)",
```

Comment:
Please add `.` to the end of the sentence, keep it consistent with previous descriptions.